### DISCUSSION TITLE
The IRC bridge is dead (again)

### DISCUSSION THREAD

**Discussion by Efruit** [space-wizards/space-station-14#5398] [General]
...and it has been for over 24hrs.

cc @PJB3005 fix your bot
also cc @mirrorcult @Zumorica bug PJB to fix it

**Comment by mirrorcult** [comment]
oh fuck

**Comment by mirrorcult** [comment]
i think this is a general MoMMI/pj issue since it's been fucky for a while

**Comment by PJB3005** [comment]
Fixed.

**Comment by gradientvera** [comment]
> @Zumorica bug PJB to fix it

a tad difficult if you ask me at 5 AM :U

**Comment by PJB3005** [comment]
You realize me and Vera are in the same time zone, right?

**Comment by PaulRitter** [comment]
the solution is quite simple, stop using technology from 2000

**Reply by PJB3005** [reply]
>2000

**Reply by moonheart08** [reply]
you mean 1988, paul?

**Reply by PaulRitter** [reply]
sorry, i highballed it

**Comment by Efruit** [comment]
hey guys guess what's down again

**Reply by Efruit** [reply]
consider: if it remains down, you all will have to triage **all those rich text bugs** by yourselves

@PJB3005
